website/content/docs/intro/usecases/what-is-a-service-mesh.mdx

```diff
@@ -0,0 +1,112 @@
+---
+layout: docs
+page_title: What is a service mesh?
+description: >-
+  Learn what a service mesh is, it's benefits, and how it works. 
+  A service mesh can solve many of the modern challenges that exist in multi-platform and multi-cloud application architectures, ranging from security to application resiliency.
+---
+
+# What is a Service Mesh?
+
+A _service mesh_ is a dedicated network layer that provides secure service-to-service communication for on-prem, cloud, or multi-cloud infrastructure.
+Service meshes are often used with a microservice architectural pattern, but can provide value in any scenario where complex networking is involved.
```

Comment:
@jkirschner-hashicorp This would be another question or headline. It would generate more traffic from those searching for "microservices architecture patterns." If this isn't too much feel free to include this.